recipes/aws-sdk-cpp/all/patches/1.9.234-0001-disable-sort-links.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cmake/sdks.cmake b/cmake/sdks.cmake
+index 0ec9981ddf..d86a9f4874 100644
+--- a/cmake/sdks.cmake
++++ b/cmake/sdks.cmake
+@@ -269,7 +269,7 @@ function(add_sdks)
+
+     # the catch-all config needs to list all the targets in a dependency-sorted order
+     include(dependencies)
+-    sort_links(EXPORTS)
++    #sort_links(EXPORTS)
+
+     # make an everything config by just including all the individual configs
+     file(WRITE ${CMAKE_BINARY_DIR}/aws-sdk-cpp-config.cmake "")
```

Comment:
It looks like cmake is happy without this patch so you might remove it for 1.9.234

Reply:
Here is the tracking issue: https://github.com/aws/aws-sdk-cpp/issues/1816

Reply:
Ok, removed